data.sql

```diff
@@ -11,3 +11,24 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
     VALUES ('Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+    VALUES ('Charmander', '2020-02-08', 0, false, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+    VALUES ('Plantmon', '2021-11-15', 2, true, 5.7);
```

Comment:
- [ ] From the data provided in the project requirement document, the weight of `Plantmon` is a negativevalue with decimal `-5.7` . Kindly update your code.